asset-transfer-private-data/chaincode-go/collections_config.json

```diff
@@ -0,0 +1,26 @@
+[
+ {
+   "name": "assetCollection",
+   "policy": "OR('Org1MSP.member', 'Org2MSP.member')",
+   "requiredPeerCount": 0,
+   "maxPeerCount": 3,
+   "blockToLive":1000000,
+   "memberOnlyRead": true
```

Comment:
Let's set memberOnlyWrite to true, implying that only these two orgs can write to the collection.